presto/base/presto-cr.yaml

```diff
@@ -103,7 +103,7 @@ spec:
       image:
         imagepullPolicy: IfNotPresent
       postgreSql:
-        jdbcUrl: "postgresql://datacatalog:datacatalog@postgres:5432/datacatalog"
+        jdbcUrl: "jdbc:postgresql://postgres.dh-dev-presto.svc.cluster.local:5432/datacatalog"
```

Comment:
It looks like this value is specific to a dev environment. Should it not go in the dev overlay?

Reply:
Moved to dev overlay